lib/al/Library/Collision/PartsConnector.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <math/seadMatrix.h>
+#include "PartsMtxConnector.h"
+
+namespace al {
+class ActorInitInfo;
+class CollisionDirector;
+class CollisionParts;
+class HitInfo;
+class HitSensor;
+class LiveActor;
+class ScreenPointTarget;
+
+class CollisionPartsConnector : public al::MtxConnector {
+private:
+    const al::CollisionParts* mCollisionParts;
+
+public:
+    CollisionPartsConnector();
+    void init(const sead::Matrix34f*, const sead::Matrix34f&, const al::CollisionParts*);
+    void getConnectingSensor() const;
+
+    bool isConnecting() const override;
+    void clear() override;
+    virtual bool isMoved() const;
+    bool isConnectInvalidCollision() const;
```

Comment:
Function order: override/virtuals before other functions.
Can be discussed, but I prefer (ctor, init, override, virtual, other) - seems to mirror function order in the binary best so far.